liquibase-core/src/main/java/liquibase/parser/core/xml/StandardNamespaceDetails.java

```diff
@@ -11,12 +11,15 @@
 public class StandardNamespaceDetails implements NamespaceDetails {
 
     public static final String GENERIC_EXTENSION_XSD = "http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd";
-    private final Pattern standardUrlPattern;
-    private final Pattern oldStandardUrlPattern;
+    private static final String STANDARD_XSD_URL_REGEX = "http://www.liquibase.org/xml/ns/dbchangelog/(dbchangelog-[\\w\\.]+.xsd)";
+    private static final Pattern STANDARD_XSD_URL_PATTERN = Pattern.compile(STANDARD_XSD_URL_REGEX);
+
+    private static final String OLD_STANDARD_XSD_URL_REGEX = "http://www.liquibase.org/xml/ns/migrator/(dbchangelog-[\\w\\.]+.xsd)";
```

Comment:
Wow, this must be a _really_ old URL, for sure pre-3.0? Interesting new fact I learned from reading this!